I/IGLWrap/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+#=
+BINARYBUILDER_RUNNER=privileged julia build_tarballs.jl --verbose --debug       
+=#
+
+using BinaryBuilder
+using Pkg
+
+version = v"2.3.0" # libIGL version, see below:
+
+sources = [
+	DirectorySource("./iglwrap"),
+	# 2.3.0 stable release:
+	GitSource("https://github.com/libigl/libigl.git",
+		"e60423e28c86b6aa2a3f6eb0112e8fd881f96777")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir -p build
+cd build
+# this line seems to prevent compilation; removed for testing:
+cmake \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_MODULE_PATH=${WORKSPACE}/srcdir/libigl/cmake\; \
```

Comment:
Based on [CMake documentation](https://cmake.org/cmake/help/v3.22/command/try_run.html#behavior-when-cross-compiling) I believe you should set something like
```suggestion
    -DCMAKE_MODULE_PATH=${WORKSPACE}/srcdir/libigl/cmake\; \
    -DCGAL_test_cpp_version_RUN_RES=0 \
    -DCGAL_test_cpp_version_RUN_RES__TRYRUN_OUTPUT="__cplusplus is $(c++ -xc++ - -E -dM < /dev/null | grep -oP '__cplusplus \K.*')" \
```